docs/pages/run/getting-started/quick-start-custom-guide.md

```diff
@@ -209,12 +209,12 @@ ls *.vars
 
 ### Select your Network
 
-Each network has specifics variables that you may want to setup for use. We will use `Holesky` to demonstrate connecting to a public testnet.
+Each network has specifics variables that you may want to setup for use. We will use `Hoodi` to demonstrate connecting to a public testnet.
 
-Open the `holesky.vars` file.
+Open the `hoodi.vars` file.
```

Comment:
https://github.com/ChainSafe/lodestar-quickstart does not support hoodi yet cc @philknows

Reply:
[Variable file added](https://github.com/ChainSafe/lodestar-quickstart/pull/64). Tested with Lodestar v1.28.0-rc.1 and Besu 25.3.0. Both with baked in configs and works. Just awaiting other EL client releases and it should be good to go.